.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

Comment:
keeps your actions up to date (some old version in here!)

Reply:
Lovely!